### PR TITLE
Bigtable: Move admin api into its own artifact.

### PR DESCRIPTION
**Pull request by igorbernstein2**
The target usecases are different enough that the clients should be split.
Also, it avoids confusion associated with duplicate static names

### CODE REVIEW

**Comment by igorbernstein2** [comment]
This is ready for review

**Comment by igorbernstein2** [comment]
@pongad Can you take a look? I would like to merge this sooner rather than later to prevent it from diverging in the next client regen

File: google-cloud-clients/google-cloud-bigtable-admin/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+[![Dependency Status](https://www.versioneye.com/user/projects/58fe4c8d6ac171426c414772/badge.svg?style=flat)](https://www.versioneye.com/user/projects/58fe4c8d6ac171426c414772)
+
+- [Product Documentation][bigtable-product-docs]
+- [Client Library Documentation][bigtable-client-lib-docs]
+
+> Note: This client is under heavy development and should not be used in production.
+
+## Quickstart
+
+[//]: # ({x-version-update-start:google-cloud-bigtable:released})
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-bigtable-admin/README.md]: this needs to be `google-cloud-bigtable-admin` so it gets the right version

File: google-cloud-clients/google-cloud-bigtable-admin/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+## License
+
+Apache 2.0 - See [LICENSE] for more information.
+
+[CONTRIBUTING]:https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/CONTRIBUTING.md
+[LICENSE]: https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/LICENSE
+[cloud-platform]: https://cloud.google.com/
+[cloud-bigtable]: https://cloud.google.com/bigtable/
+[bigtable-product-docs]: https://cloud.google.com/bigtable/docs/
+[bigtable-client-lib-docs]: https://googlecloudplatform.github.io/google-cloud-java/google-cloud-clients/apidocs/index.html?com/google/cloud/bigtable/package-summary.html
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-bigtable-admin/README.md]: This needs to point to the admin docs

File: google-cloud-clients/google-cloud-bigtable-admin/pom.xml [hunk 3]
```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-bigtable-admin</artifactId>
+  <version>0.55.2-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigtable:current} -->
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-bigtable-admin/pom.xml]: Update the version-update string to the right artifact

File: google-cloud-clients/pom.xml [hunk 4]
```diff
@@ -414,6 +414,7 @@
   <modules>
     <module>google-cloud-automl</module>
     <module>google-cloud-bigtable</module>
+    <module>google-cloud-bigtable-admin</module>
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/pom.xml]: You also need to update the Stub list further down

File: utilities/generate_api.py [hunk 5]
```diff
@@ -30,7 +30,6 @@
 
 
 dir_overrides = {
-    'bigtable-admin': 'google-cloud-bigtable',
     'error-reporting': 'google-cloud-errorreporting',
```
Comment (garrettjonesgoogle) [inline utilities/generate_api.py]: Could you add bigtable-admin to the batch generation script?

File: google-cloud-clients/google-cloud-bigtable-admin/README.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
+# Google Cloud Java Client for Bigtable Admin
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-bigtable-admin/README.md]: Add a link to google-cloud-bigtable-admin from the root README

File: google-cloud-clients/google-cloud-bigtable-admin/README.md (hunk 1 above)
Comment (igorbernstein2) [reply google-cloud-clients/google-cloud-bigtable-admin/README.md]: done

File: google-cloud-clients/google-cloud-bigtable-admin/README.md (hunk 2 above)
Comment (igorbernstein2) [reply google-cloud-clients/google-cloud-bigtable-admin/README.md]: done

File: google-cloud-clients/google-cloud-bigtable-admin/pom.xml (hunk 3 above)
Comment (igorbernstein2) [reply google-cloud-clients/google-cloud-bigtable-admin/pom.xml]: done

File: google-cloud-clients/pom.xml (hunk 4 above)
Comment (igorbernstein2) [reply google-cloud-clients/pom.xml]: Not sure if I did it correctly. The list already included
`com.google.cloud.bigtable.admin.v2.stub:com.google.cloud.bigtable.data.v2.stub`

but `com.google.cloud.bigtable.data.v2.stub` had a `*`, which I removed

File: utilities/generate_api.py (hunk 5 above)
Comment (igorbernstein2) [reply utilities/generate_api.py]: It's already there. This client was already being generated, but was copied to a different directory

File: google-cloud-clients/google-cloud-bigtable-admin/README.md (hunk 6 above)
Comment (igorbernstein2) [reply google-cloud-clients/google-cloud-bigtable-admin/README.md]: done

File: utilities/generate_api.py (hunk 5 above)
Comment (garrettjonesgoogle) [reply utilities/generate_api.py]: Oh right! Sounds good.

File: google-cloud-clients/pom.xml (hunk 4 above)
Comment (garrettjonesgoogle) [reply google-cloud-clients/pom.xml]: The `*` actually needs to stay, because of the subpackages `mutaterows` and `readrows`

Comment (igorbernstein2) [reply google-cloud-clients/pom.xml]: fixed

**Comment by igorbernstein2** [comment]
Thanks!

**Pull request merged**